schema/classes/Contributor.yaml

```diff
@@ -0,0 +1,45 @@
+"@context":
+  schema: http://schema.org/
+  rdf: http://www.w3.org/1999/02/22-rdf-syntax-ns#
+  rdfs: http://www.w3.org/2000/01/rdf-schema#
+  xsd: http://www.w3.org/2001/XMLSchema#
+  owl: http://www.w3.org/2002/07/owl#
+  void: http://rdfs.org/ns/void#
+  dcterms: http://purl.org/dc/terms/
+  dcat: http://www.w3.org/ns/dcat#
+  dcmitype: http://purl.org/dc/dcmitype/
+  dandi: http://schema.dandiarchive.org/
+  prov: http://www.w3.org/ns/prov#
+  crypto: http://id.loc.gov/vocabulary/preservation/cryptographicHashFunctions/
+  md5: crypto:md5
+  sha1: crypto:sha1
+  sha256: crypto:sha256
+  sha512: crypto:sha512
+  attributedTo: prov:wasAttributedTo
+"@graph":
+  - "@id": dandi:Contributor
+    "@type":
+    - rdfs:Class
+    - prov:Agent
+    http://www.w3.org/2002/07/owl#equivalentClass:
+      - "@id": http://xmlns.com/foaf/0.1/Person
```

Comment:
Must be friday, but I have difficulty parsing this yaml structure. Funny thing it loads fine, but I wonder if it is what was expected it to be:
```
$> python -c 'import yaml; from pprint import pprint; pprint(yaml.load(open("Contributor.yaml"), Loader=yaml.BaseLoader)["@graph"])'
[{u'@id': u'dandi:Contributor',
  u'@type': [u'rdfs:Class', u'prov:Agent'],
  u'http://www.w3.org/2002/07/owl#equivalentClass': [{u'@id': u'http://xmlns.com/foaf/0.1/Person'}],
  u'rdfs:comment': u'A contributor, a person or organization, to a project.',
  u'rdfs:label': u'Contributor',
  u'rdfs:subClassOf': {u'@id': u'schema:Thing'}},
 {u'@id': u'schema:identifier',
  u'schema:domainIncludes': u'dandi:Contributor',
  u'schema:rangeIncludes': [u'dandi:Person', u'dandi:Organization']},
 {u'@id': u'schema:roleName',
  u'schema:domainIncludes': u'dandi:Contributor',
  u'schema:rangeIncludes': u'dandi:RoleType'},
 {u'@id': u'dandi:awardNumber',
  u'@type': u'rdf:Property',
  u'schema:domainIncludes': u'dandi:Contributor',
  u'schema:rangeIncludes': u'schema:Text'},
 {u'@id': u'schema:schemaVersion', u'schema:domainIncludes': u'dandi:Person'}]

```

Reply:
don't worry about those yaml files right now. they haven't been updated since our iterations on models.py. i will get back to revising/fixing all of this when i get back next week.